shared/src/androidMain/kotlin/com/bunbeauty/shared/data/FirebaseAuthRepository.kt

```diff
@@ -1,17 +1,9 @@
 package com.bunbeauty.shared.data
 
-import com.google.firebase.auth.FirebaseAuth
+actual class FirebaseAuthRepository {
```

Comment:
Lets remove if we don't need it

Reply:
We need it until update in iOS, it's `actual class`